### DISCUSSION TITLE
FR : Adding pictures/photos as an alternative for adding recipies

### DISCUSSION THREAD

**Discussion by kuhnino** [mealie-recipes/mealie#2311] [Feature Request]
I often take pictures out of magazines and old cookbooks

Adding them in an easy way to mealie would help. 
Maybe as a possibilities to add a picture in the NOTES files so that this is displayed instead of the HTML content of ingredients and tasks

I´m aware of all disadvantages that a recipie is then not searchable or editable, but this would expand the featureset a lot and the useage is boosted I guess

**Comment by michael-genson** [comment]
You can embed images in steps or in the notes (just drag and drop, or use HTML tags)

FYI you can also use OpenAI to parse recipe images now: https://github.com/mealie-recipes/mealie/pull/3974
